deployment/7_update_ens.js

```diff
@@ -0,0 +1,71 @@
+/* global artifacts */
```

Comment:
Maybe rename this file `8_update_ens.js` so it is decoupled from the wallet upgrade?